collective.cpp

```diff
@@ -761,7 +761,7 @@ vector<Vec2> Collective::getBedPositions(const vector<PCreature>& creatures, con
   optional<SquareType> bedType = getDormInfo()[spawnType].getBedType();
   vector<Vec2> bedPos;
   for (int i : All(creatures))
-    for (int j : Range(30)) {
+    for (int __attribute__((unused)) j : Range(30)) {
```

Comment:
Yeh, I've seen a few around including MSVC compatible ones. At least they it's just a find replace now if that's what you want to do.